packages/app/src/components/Bookmarks/BookmarkFolderItem.tsx

```diff
@@ -236,7 +319,7 @@ const BookmarkFolderItem: FC<BookmarkFolderItemProps> = (props: BookmarkFolderIt
             </div>
             {hasChildren() && (
               <div className="grw-foldertree-count-wrapper">
-                <CountBadge count={ childCount } />
+                <CountBadge count={childCount} />
```

Comment:
## Now
- Only folders one level down are counted
- Bookmark pages are not counted

![image](https://user-images.githubusercontent.com/68407388/216901059-8940c061-b9bd-40b0-be9a-bcc63ae06a9f.png)

## Request
- Is it possible to count and display only the number of all bookmarked pages under the distribution?
- Folders does not count.

Reply:
If you can impl that I will create new task.

Reply:
Yes it's possible to count all bookmarked pages

Reply:
I will create new task.